Library/Formula/createrepo.rb

```diff
@@ -0,0 +1,10 @@
+class Createrepo < Formula
+  homepage ""
+  url "http://createrepo.baseurl.org/download/createrepo-0.10.tar.gz"
+  version "0.10"
+  sha256 "d5ccd5ed8e7db48e6a4d60c1e415b6fe7e85a357dff28a7a25683dc9bb2b9dc9"
+
+  def install
+    system "make", "install"
+  end
+end
```

Comment:
Running "createrepo" on a directory of *.rpm packages should work as a test. If successful, it should create a "repodata" directory with some xml files in it. (Or sqlite, for the new format.)
However, it does require a bunch of python dependencies (rpm/yum, libxml2, urlgrabber, yum-metadata-parser) not listed here. So it probably doesn't work, without a more complex formula.